src/stackanalysismodule.ts

```diff
@@ -45,7 +45,7 @@ export module stackanalysismodule {
 	get_stack_metadata = (context, file_uri, cb) => {
 
         let payloadData : any;
-        let projRootPath: string = vscode.workspace.rootPath;
+        let projRootPath: string = vscode.workspace.workspaceFolders[0].uri.path;
```

Comment:
this will fail in case of user is working on the manifest for worksapace2 as `let projRootPath: string = vscode.workspace.workspaceFolders[0].uri.path;` projRootPath will always be of first one